styles/global-styles.scss

```diff
@@ -1,3 +1,9 @@
 body {
   font-family: 'Roboto', sans-serif;
 }
+
+.info-repo {
```

Comment:
The styles look good, just create a separate Footer component and import the styles in the js file.